config/initializers/highlight_override.rb

```diff
@@ -0,0 +1,54 @@
+# Monkey patch the generated Bindings for the swagger Highlight model
```

Comment:
When we make v1 we’ll have to put the v1 override somewhere.  If our scripts don’t blow away the bindings directory, would you want to put this file directly in `/app/bindings/api/v0/bindings`?